tests/integration/profile_update/test_bookmark_profile_update.py

```
@@ -0,0 +1,437 @@
+"""
+북마크 프로필 업데이트 통합 테스트
+
+북마크 이벤트 리스너와 사용자 프로필 처리기의 통합 테스트
+"""
+
+import pytest
+from unittest.mock import AsyncMock, MagicMock, patch
+import asyncio
+from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)